cmd/oci/main.go

```diff
@@ -27,6 +30,12 @@ import (
 var version string
 var build string
 
+// All registered drivers.
+var (
```

Comment:
Since we already know the drivers ahead of time it seems a complex way to do this. Why do we need to have a mutex and a mutable map of drivers? Could we not just create a static map?

```
drivers = map[string]flexvolume.Driver{
"oci-bvs": driver.OCIFlexvolume,
}
```